doc/motr-kv-app.md

```diff
@@ -72,6 +72,11 @@ The steps to delete an existing index: function index\_delete().
 *   Retrieve the result.
 *   Finalize and free the operation with m0\_op\_fini() and m0\_op\_free().
 
+## Troubleshoot
+
+To find the application's 2nd parameter (LOCAL_ADDR), look for the running motr's Client_addr. 
```

Comment:
This needs more details.
How?
Which command to run?
What is the sample output?
Which part of the output can be taken as the LOCAL_ADDR?

Reply:
Sample command and outputs are added, with parameter details.